### PR TITLE
Meals QR Scan & API functionallity. 

### PR DESCRIPTION
**Pull request by EncryptEx**
## Patch Notes
Created / Made the HTML page to scan QR codes at any meal of the event. 
JavaScript's background added.
POST requests done.
Fixed issues with CSRF's tokens and session IDs

## Additional Notes (optional)

Would need to fix and error of duplication of instances of ``babel-polyfill`` Caused by bootstrap and instascan (library used)

## Some questions
- [x] I have read the contributing guidelines
- [x] I abide by this repository Code of Conduct
- [x] I understand that my PR won't be merged until GitHub gives a "green light"

### CODE REVIEW

File: meals/templates/meal_checkin.html [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+    
+    <script type="text/javascript" src="https://rawgit.com/schmich/instascan-builds/master/instascan.min.js"></script>
 
 {% endblock %}
 
 {% block extra_scripts %}
     <script src="{% static 'lib/instascan.min.js' %}"></script>
-    <script type="text/javascript" src="{% static 'js/meals_checkin.js' %}"></script>
 
 {% endblock %}
```
Comment (Casassarnau) [inline meals/templates/meal_checkin.html:15]: Si borres la linia de sota torna la navbar

File: meals/templates/meal_checkin.html [hunk 2]
```diff
@@ -20,14 +20,167 @@
...
 <div class="container-fluid full-screen">
     <div class="row full-screen">
-        <div class="col-md-offset-3 col-md-6 col full-screen">
-            <div id="meals-scan-image" class="meals-inside-scan" style="max-width: 100%;max-height: 100%;" data-mealid="{{ meal.id }}"
-            data-apiurl="{% url 'cool_meals_api' %}">
-                <a class="btn btn-default btn-block" href="{% url 'meals_list' %}">< Back</a>
-                <video id="meals-scan-video" style="width: 100%; max-height: 80%; margin:0 auto;">   </video>
-                <select class="form-control"></select>
+        
+            <a class="btn btn-default btn-block" href="{% url 'meals_list' %}">< Back</a>
```
Comment (Casassarnau) [inline meals/templates/meal_checkin.html:24]: Aixo a poder ser estar dins de la div de sota i  sota de tot per a que es vegi a sota el video, seria molt top

**Review by Casassarnau — CHANGES_REQUESTED** [review]

File: meals/templates/meal_checkin.html (hunk 1 above)
Comment (EncryptEx) [reply meals/templates/meal_checkin.html:15]: (https://github.com/hackupc/myhackupc/blob/474739e2a21210344c23ac6f6c6fce4f436fe719/meals/templates/meal_checkin.html#L16)

**Review by Casassarnau — APPROVED** [review]
Nice

**Pull request merged**